app/templates/views/organisations/organisation/index.html

```diff
@@ -112,4 +112,10 @@ <h3 class="govuk-!-font-weight-bold govuk-!-font-size-24 govuk-!-margin-bottom-3
     <div class="keyline-block govuk-!-margin-top-2"></div>
   {% endif %}
 
+  <div class="js-stick-at-bottom-when-scrolling">
+    <p class="bottom-gutter">
+      <a href="{{ download_link }}" download="download" class="govuk-link govuk-link--no-visited-state govuk-!-font-weight-bold">Download this report</a>
+    </p>
```

Comment:
It’s not consistent with other sticky footer links, but I think here the link looks best bold and left-aligned. It just needs the spacing tweaking slightly so there’s equal space above and below the link:
```suggestion
    <p class="govuk-!-margin-bottom-1">
      <a href="{{ download_link }}" download="download" class="govuk-link govuk-link--no-visited-state govuk-!-font-weight-bold">Download this report</a>
    </p>
```

It’ll look like this:
![image](https://user-images.githubusercontent.com/355079/142402803-083f261f-6257-48f4-8689-c342c21fd4c0.png)